src/components/Icon/Icon.stories.tsx

```diff
@@ -116,8 +123,9 @@ const transactionFor = (
   direction: TransactionDirection,
```

Comment:
i think we can put this directly inline with the component

Reply:
I migrated to the mock and put it inline